client/src/Components/CommonComponent/Table/index.js

```diff
@@ -13,36 +13,34 @@ const Table = (data, id) => {
         </tr>
       </thead>
       <tbody className="studentTable__content">
-        {data.map(e => {
-          return (
-            <tr>
-              <Router>
-                <td className="studentTable__content1">
-                  {e.name}
-                  <p>
-                    <Link
-                      to={`/profile/teacher/:${id}`}
-                      className="studentTable__link"
-                    >
-                      view profile
-                    </Link>
-                  </p>
-                </td>
-                <td className="studentTable__content2">
-                  {e.subject}
-                  <p>
-                    <Link
-                      to={`/student/:subject/:${id}	`}
-                      className="studentTable__link"
-                    >
-                      view subject
-                    </Link>
-                  </p>
-                </td>
-              </Router>
-            </tr>
-          );
-        })}
+        {data.map(e => (
+          <tr>
+            <Router>
+              <td className="studentTable__content1">
+                {e.name}
+                <p>
+                  <Link
+                    to={`/profile/teacher/:${e.id}`}
```

Comment:
btw, I know this is not your changes, but it is good to have these URLs defined in a separate file as constants/functions instead of hardcoded strings like the one here, so that in case the URL changes in the future, you have just one place to update and don't have to search through the whole codebase...

Reply:
OK, I will try your suggestion.
thanks